test/cli/harness/node.go

```diff
@@ -96,6 +97,38 @@ func (n *Node) UpdateConfig(f func(cfg *config.Config)) {
 	n.WriteConfig(cfg)
 }
 
+func (n *Node) ReadUserSuppliedRessourceOverrides() *rcmgr.PartialLimitConfig {
```

Comment:
This is a mouthful, I'd use something like "ReadLimitsJSON".  Just a nit, feel free to ignore.

Reply:
Other (nit):
1. Resource instead of Ressource
2. Lets use whatever name we arrive at for this concept above (e.g., UserResourceLimitOverrides)

Reply:
I've changed to `ReadUserResourceOverrides`